package.json

```diff
@@ -19,16 +19,16 @@
   },
   "homepage": "https://github.com/voyagegroup/eslint-config-fluct",
   "peerDependencies": {
-    "eslint": "^4.0.0"
+    "eslint": "4.0.0"
   },
   "optionalDependencies": {
-    "eslint-plugin-node": "^5.1.0",
-    "eslint-plugin-react": "^7.1.0"
+    "eslint-plugin-node": "^5.1.1",
+    "eslint-plugin-react": "^7.2.1"
   },
   "devDependencies": {
-    "eslint": "^4.0.0",
-    "eslint-plugin-markdown": "^1.0.0-beta.3",
-    "eslint-plugin-node": "^4.2.0",
-    "eslint-plugin-react": "^7.1.0"
+    "eslint": "4.0.0",
```

Comment:
Why lock this field?

This field specifies packages we use in CI to test changesets. And we specify more detailed versions in yarn.lock. We need not to lock this version without the case that we face some serious regressions in a newer version. Are there any problems in CI?